logstreams/src/test/java/io/zeebe/logstreams/state/ReplicateSnapshotControllerTest.java

```diff
@@ -167,69 +159,44 @@ public void shouldNotFailOnReplicatingAndReceivingTwice() throws Exception {
     assertThat(valueFromSnapshot).isEqualTo(VALUE);
   }
 
-  @Test
-  public void shouldEnsureMaxSnapshotCount() throws Exception {
-    // given
-    receiverSnapshotController.consumeReplicatedSnapshots();
-    replicateXSnapshots(3);
-
-    // when
-    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-
-    // then
-    final RocksDBWrapper wrapper = new RocksDBWrapper();
-    final long recoveredSnapshot = receiverSnapshotController.recover();
-    assertThat(recoveredSnapshot).isEqualTo(3);
-
-    wrapper.wrap(receiverSnapshotController.openDb());
-    final int valueFromSnapshot = wrapper.getInt(KEY);
-    assertThat(valueFromSnapshot).isEqualTo(VALUE);
-
-    assertThat(receiverStorage.existSnapshot(1)).isFalse();
-    assertThat(receiverStorage.existSnapshot(2)).isTrue();
-    assertThat(receiverStorage.existSnapshot(3)).isTrue();
-  }
-
-  @Test
-  public void shouldInvokeCallbackOnReplicatedSnapshot() {
-    // given
-    receiverSnapshotController.consumeReplicatedSnapshots();
-    final NoopDeletionService mockDeletionService = spy(new NoopDeletionService());
-    receiverSnapshotController.setDeletionService(mockDeletionService);
-    replicateXSnapshots(3);
-
-    // then
-    verify(mockDeletionService, never()).delete(anyInt());
-
-    // when
-    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-
-    // then
-    verify(mockDeletionService).delete(2);
-  }
-
-  private void replicateXSnapshots(final int snapshotAmount) {
-    for (int i = 1; i <= snapshotAmount; ++i) {
-      replicatorSnapshotController.takeSnapshot(i);
-      replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-    }
-  }
+  //  @Test
+  //  public void shouldEnsureMaxSnapshotCount() throws Exception {
+  //    // given
+  //    receiverSnapshotController.consumeReplicatedSnapshots();
+  //    replicateXSnapshots(3);
+  //
+  //    // when
+  //    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
+  //
+  //    // then
+  //    final RocksDBWrapper wrapper = new RocksDBWrapper();
+  //    final long recoveredSnapshot = receiverSnapshotController.recover();
```

Comment:
This is also out commented

Reply:
Removed since this was moved to the broker with the snapshot store/storage impl